target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -247,6 +247,19 @@ define Device/redmi_ax6
 endef
 TARGET_DEVICES += redmi_ax6
 
+define Device/spectrum_sax1v1k
+	$(call Device/FitImage)
+	$(call Device/EmmcImage)
```

Comment:
`Device/EmmcImage` produce `factory.bin` image that is not used at all.
You can leave this call and add:
    `IMAGES := sysupgrade.bin`
or remove this call and add:
    `IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-to 64k | sysupgrade-tar rootfs=$$$$@ | append-metadata`

Reply:
I referenced qnap 301w which is ipq8072 with emmc
```
define Device/qnap_301w
	$(call Device/FitImage)
	$(call Device/EmmcImage)
	DEVICE_VENDOR := QNAP
	DEVICE_MODEL := 301w
	DEVICE_DTS_CONFIG := config@hk01
	KERNEL_SIZE := 16384k
	SOC := ipq8072
	DEVICE_PACKAGES := ipq-wifi-qnap_301w
endef
TARGET_DEVICES += qnap_301w
```

Reply:
I'm not sure if Qnap uses the factory image. But you don't have to generate an unused image.

Reply:
fixed